### PR TITLE
Use existing binaries for Rust tools, where possible

### PR DESCRIPTION
**Pull request by anghelcovici**
# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -195,7 +195,26 @@ RUN rustup component add \
...
   rustfmt
 
+# No binary available on Github, have to use cargo install
 RUN cargo install cargo-deadlinks
-RUN cargo install cargo-deny
-RUN cargo install cargo-crev
-RUN cargo install grcov
+
+# Where to install rust tooling
+ARG install_dir=${rustup_dir}/bin
```
Comment (tiziano88) [inline Dockerfile:202]: Maybe we should have an overall `tools_dir` and use it for most single-binary tools?

Comment (anghelcovici) [reply Dockerfile:202]: I did initially that. But then I noticed that this is where the tools went initially, and as these were rust based, so it seemed natural.
Usually the tools install in `/usr/local/bin` or `/usr/bin`, but we could have another folder.

**Comment by anghelcovici** [comment]
This is actually quite horrible.. again problems with users in Docker. If I install cargo-crev it works to run it, but any operation fails, and I think is because we install and run rustup as root. I'll try to see if there is any workaround

**Comment by tiziano88** [comment]
> This is actually quite horrible.. again problems with users in Docker. If I install cargo-crev it works to run it, but any operation fails, and I think is because we install and run rustup as root. I'll try to see if there is any workaround

I think you just need to change the invocation from `cargo crev repo fetch url https://github.com/dpc/crev-proofs` to `cargo-crev crev repo fetch url https://github.com/dpc/crev-proofs` (there is a missing `crev` currently)

**Pull request merged**